R/topics.R

```diff
@@ -167,3 +167,56 @@ topic_must <- function(message, topic) {
     x = paste0("Not ", encodeString(topic, quote = "'"))
   ))
 }
+
+content_info <- function(content_entry, index, pkg, section) {
+
+  if (grepl(".*::.*", content_entry)) {
+    names <- strsplit(content_entry, "::")[[1]]
+    if (!rlang::is_installed(names[1])) {
+      abort(
+        sprintf(
+          "%s must be installed if it is mentioned in the reference index.",
+          names[1]
+        )
+      )
+    }
+
+    path <- downlit::href_topic(names[2], names[1])
+    if (is.na(path)) {
+      abort(
+        sprintf(
+          "Could not find an href for topic %s of package %s",
+          names[2], names[1]
+        )
+      )
+    }
+
+    rd_name <- fs::path_ext_set(fs::path_file(path), "Rd")
+    file <- withr::local_tempfile(fileext = ".html")
+    write_lines(capture.output(tools::Rd2HTML(tools::Rd_db(names[1])[[rd_name]])), file)
+    rd <- xml2::read_html(file)
+
+
+    tibble::tibble(
+      path = path,
+      aliases = content_entry,
```

Comment:
Should it just be the function name? Presumably you're doing this because you've re-exported the function so it looks weird to have the package name?

Reply:
I've made a new proposal "function (from package)" as I think it might puzzle the user to get to another website when clicking?